.github/workflows/tag-and-create-release.yml

```
@@ -0,0 +1,56 @@
+      release_type:
+        type: string
+        description: 'Release type'
+        required: true      
+      dotnet-version:
+        type: string
+        required: true
+        description: Version of .NET to use (e.g., 8.0.x)
+    outputs:
+      VersionNumber:
+        description: "The version number generated"
+        value: ${{ jobs.bump_version_and_generate_release_notes.outputs.VersionNumber }}
```

actionlint (1.7.7)

property "versionnumber" is not defined in object type {}

(expression)